client/src/login/styles/login.scss

```diff
@@ -0,0 +1,61 @@
+.login-page {
+  justify-content: center;
+  width: 100%;
+  max-width: 27rem;
+  min-width: 0;
+  margin-top: 1.5rem;
+  @include forSize('phoneOnly') {
+    margin-top: 4.5rem;
+  }
+  @include forSize('tabletPortraitUp') {
+    margin-top: 0;
+    max-width: 28.375rem;
+  }
+  @include forSize('tabletLandscapeUp') {
+    max-width: 49.125rem;
+  }
+  @include forSize('desktopUp') {
+    max-width: 52.5rem;
+  }
+}
+
+.login-page__text {
+  margin-bottom: 3rem;
+}
+
+.login-page__buttons {
+  display: flex;
+  align-items: center;
+}
+
+.login-page__buttons--orcid {
+  margin-right: 1.5rem;
+}
+
+.login-page__image {
+  width: 75%;
+  height: 100%;
+  display: none;
+  margin-left: auto;
+  @include forSize('tabletPortraitUp') {
+    display: block;
+  }
+}
+
+.login-page__image--image {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+.login-page__image--credit {
+  color: $colourTextLarge;
+  font-size: 12px;
+  text-align: right;
+  //padding-left: 0.75rem; // not sure about the padding, added it because it was in the old styles but it make the text mis-aligned
```

Comment:
Can we remove this commented out code if the styling works without it?

Reply:
oops